packages/ui/src/components/approver/animations/approver-animation.web.tsx

```diff
@@ -1,6 +1,6 @@
 import { motion, stagger, useAnimate } from 'framer-motion';
 import { css } from 'leather-styles/css';
-import { HasChildren } from 'src/utils/has-children';
+import { HasChildren } from 'src/utils/has-children.shared';
```

Comment:
Can you rebase on dev and make sure there are no `src/` imports? Think they should cause type errors on rebase

Reply:
I rebased and cleaned this up. I think we should add path aliases to the `tsconfig` to clean up all of the relative path imports. Is there a reason not to do that? 

It would mean we need to have the `src/` path in `ui/tsconfig.json`:

```
"paths": {
      "leather-styles/*": ["leather-styles/*"],
      "@components/*": ["src/components/*"],
      "@icons/*": ["src/icons/*"],
      "@hooks/*": ["src/hooks/*"],
      "@utils/*": ["src/utils/*"],
```